src/entities/User/model.ts

```diff
@@ -45,10 +45,11 @@ export default class UserModel extends Model<UserAttributes> {
     return await this.namedQuery('get_addresses_by_forum_ids', query)
   }
 
-  static async getDiscordIdsByAddresses(
-    addresses: string[],
-    shouldDiscordBeActive = true
-  ): Promise<ValidatedDiscordAccount[]> {
+  static async getActiveDiscordIds(addresses: string[]): Promise<ValidatedDiscordAccount[]> {
+    return await this.getDiscordIds(addresses, true)
+  }
+
+  static async getDiscordIds(addresses: string[], shouldDiscordBeActive = false): Promise<ValidatedDiscordAccount[]> {
```

Comment:
Why drop `ByAddresses`?

Reply:
I wanted to put the focus on the distinction between active discord ids, and all discord ids (which was kind of hidden in a default param)

Also, I didn't think we need that anymore, because now the only parameter is called `addresses` (and there is no other way of getting them).